vcd/resource_vcd_org_vdc.go

```diff
@@ -201,7 +201,18 @@ func resourceVcdOrgVdc() *schema.Resource {
 				Optional:    true,
 				Description: "True if discovery of vCenter VMs is enabled for resource pools backing this VDC. If left unspecified, the actual behaviour depends on enablement at the organization level and at the system level.",
 			},
-
+			"elasticity": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Set to true to indicate if the Flex vDC is to be elastic.",
```

Comment:
"vDC" should be "VDC" in user-facing messages. There are more places in this PR where this should be adjusted.

Reply:
changed